forem/forem
# GIF button z-index issue

## What type of PR is this? (check all applicable)


## Description

This PR fixes z-index issue of "gif play/pause" button.

## Related Tickets & Documents

- Closes #16643

## QA Instructions, Screenshots, Recordings

1. Go to any post having gif image
2. There is a play/pause button on the GIF
3. When you scroll down, the button should stay BELOW the header - it should not overlay it.

## Added/updated tests?

No

## [Forem core team only] How will this change be communicated?

It won't.
